.github/workflows/_run-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Github Actions Tests
+
+on:
+  workflow_call:
+    secrets:
+      pact_broker_username:
+        required: true
+        description: required if `publish_pacts` is `true`
```

Comment:
There's isn't a separate public_pacts, I think you can remove this description (and from the one below)

Reply:
Removed